tools/wpt/testharnessreport.js

```diff
@@ -9,5 +9,5 @@ window.add_result_callback(({ message, name, stack, status }) => {
 });
 
 window.add_completion_callback((tests, harnessStatus) => {
-  Deno.exit(0);
+  window.close();
```

Comment:
This is basically the same as
```ts
queueMicrotask(() => {
  setTimeout(() => {
    Deno.exit(0);
  }, 0);
});
```
And I observed some flaky test failure with this change with this test case on my local machine (mac) https://github.com/denoland/wpt/blob/master/html/webappapis/timers/negative-settimeout.any.js#L3

But it might work in CI. I'm ok with landing this.

Reply:
Ah. I understand why it is flakey now... I can fix the upstream WPT test to not be flakey.